File: solutions/go/raindrops/1/raindrops.go
package raindrops

import (
	"fmt"
)

func Convert(number int) string {

	var x string

	if number%3 == 0 {
		x += "Pling"
	}
	if number%5 == 0 {
		x += "Plang"
	}
	if number%7 == 0 {
		x += "Plong"
	}
	if number%3 != 0 && number%5 != 0 && number%7 != 0 {
		x += fmt.Sprint(number)
	}

	return fmt.Sprint(fmt.Sprint(x))
}

File: solutions/c/raindrops/1/raindrops.c
#include "raindrops.h"
#include <stdio.h>
#include <string.h>

void convert(char result[], int drops) {
    result[0] = '\0';

    if (drops % 3 == 0)
        strcat(result, "Pling");
    if (drops % 5 == 0)
        strcat(result, "Plang");
    if (drops % 7 == 0)
        strcat(result, "Plong");

    if (result[0] == '\0')
        snprintf(result, BUFFER_LENGTH, "%d", drops);
}